pkg/kn/commands/service/export.go

```diff
@@ -0,0 +1,240 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"errors"
+
+	"sort"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"knative.dev/client/pkg/kn/commands"
+	clientservingv1alpha "knative.dev/client/pkg/serving/v1"
```

Comment:
@rhuss has gone through renaming imports exercise a while ago, it should be the following:
```suggestion
	clientservingv1 "knative.dev/client/pkg/serving/v1"
```

Reply:
Changed. Thank you!